lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -110,47 +125,52 @@ class _FindClosestFacilityFromPointSampleState
       geometry2: incidentsLayer.fullExtent!,
     );
 
+    // Set the viewpoint geometry on the map view controller
     _mapViewController.setViewpointGeometry(mapExtent, paddingInDiPs: 30);
 
+    // Generate the closest facility parameters
     _closestFacilityParameters = await generateClosestFacilityParameters(
         facilitiesLayer, incidentsLayer);
 
-    setState(() => _isInitialized = true);
+    // Set the initialized flag to true
+    setState(() => _initialized = true);
```

Comment:
I think we should make this `_ready` like other samples, and add the progress indicator pattern. We could also then remove the `_isInitialized` check off line 80.